File: C2002LDoiCan/APC#/06-07-2021/exam3/exam3/Circle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace exam3
{
    class Circle : Shape
    {
        private double radius;
        public double Radius { 
            get {
                return radius;
            }
            set {
                radius = value;
            } 
        }
        public override void ShowArea()
        {
            Console.WriteLine("Area = {Math.Pow(radius, 2) * Math.PI}");
        }
    }
}

File: C2009G/adf1/08-06-2021/src/main/java/models/Circle.java
package models;

public class Circle extends Shape{
    private Float radius;
    @Override
    //Polimophism
    public Float getArea() {
        return (float)Math.PI * radius * radius;
    }

    @Override
    public Float getPerimeter() {
        return (float)Math.PI * 2 * radius;
    }
}
